### PR TITLE
Add Almalinux9 AARCH64 and Rocky9 AARCH64

### PR DESCRIPTION
**Pull request by smahima27**

### CODE REVIEW

**Comment by smahima27** [comment]
@bastelfreak I am trying to add arm support for almalinux 9 and rocky 9. Tests are failing can you help with this one?

**Comment by bastelfreak** [comment]
Hi @smahima27 , I don't have an aarch64 test setup at the moment so I cannot test this :(

**Comment by smahima27** [comment]
@bastelfreak fixed the issue. Can you please review?

File: lib/beaker-hostgenerator/data.rb [hunk 1]
```diff
@@ -1139,6 +1139,7 @@ def generate_osinfo
       %w[almalinux rocky].each do |os|
         (8..9).each do |release|
           yield ["#{os}#{release}-64", "el-#{release}-x86_64"]
+          yield ["#{os}#{release}-AARCH64", "#{os}#{release}-aarch64"]
```
Comment (bastelfreak) [inline lib/beaker-hostgenerator/data.rb:1142]: I'm not really sure how I can verify that this is correct

Comment (mhashizume) [reply lib/beaker-hostgenerator/data.rb:1142]: I think we'll want to stay consistent between x86_64 and AARCH64 in terms of using the EL as the platform for Alma and Rocky. Could you update to 

```
          yield ["#{os}#{release}-AARCH64", "el-#{release}-aarch64"]
```

Comment (smahima27) [reply lib/beaker-hostgenerator/data.rb]: made the changes.

**Review by bastelfreak — APPROVED** [review]

**Comment by smahima27** [comment]
@bastelfreak need your help to merge the PR. Due to lack of permissions.

**Pull request merged**